sql.ipynb

```diff
@@ -0,0 +1,285 @@
+{
```

Comment:
<p>Is there a clean way to get example code blocks that can't be run? Obviously this will throw an error in the binder, but it would be nice to have an example with syntax highlighting for people to look at.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dask/dask-examples/pull/209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
You could just do a markdown cell with a code fence inside it

````markdown

# Your markdown

```python
import foo

print("hello world")
```

````

Reply:
I did this ins the `Remote Clusters SSH` section of this notebook because the examples weren't working.

https://github.com/jacobtomlinson/dask-video-tutorial/blob/main/08-Distributed.ipynb

Reply:
Cool - thanks for the example